src/styles/header.scss

```diff
@@ -0,0 +1,68 @@
+@import './variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: white;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  .logo-link {
+    line-height: 0;
+  }
+
+  .nav {
+    display: flex;
+    margin: 0;
+
+    ul {
```

Comment:
Use only classes selectors for styles, fix it everywhere

Reply:
if i change it the code doesn't work